test/cli/t/add-del.t

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+# a TAP compatible test
+
+# quick and dirty: fail early
+set -eu
+
+# test plan
+echo 1..8
+
+# usage:
+#  foo || tapfail
+#  echo "ok"
+# this will tell TAP "not ok" IFF foo has failed
+tapfail() {
+  echo -n "not "
+}
+
+YAST=/usr/sbin/yast
+
+echo "# assume and check that eth0 exists, as a base for the vlan"
+ip addr show dev eth0 || tapfail
+echo "ok 1 eth0: exists already"
```

Comment:
Yes, `not ok 1 eth0: exists already` is according to http://testanything.org/ . I hope it will be a little less confusing when I insert a dash after the test number.